internal/network/command.go

```diff
@@ -386,3 +393,19 @@ func addGatewayFlag(cmd *cobra.Command, c *pcmd.AuthenticatedCLICommand) {
 		return autocompleteGateways(c.V2Client, environmentId)
 	})
 }
+
+func formatZoneInfoItems(zoneInfoItems []networkingv1.NetworkingV1ZoneInfo) ([]string, error) {
+	var formattedItems []string
+
+	for _, item := range zoneInfoItems {
+		if item.ZoneId != nil && item.Cidr != nil {
+			formattedItems = append(formattedItems, fmt.Sprintf("%s=%s", *item.ZoneId, *item.Cidr))
+		} else if item.Cidr != nil {
+			formattedItems = append(formattedItems, *item.Cidr)
+		} else {
+			return []string{}, fmt.Errorf("zone info item is missing CIDR")
+		}
```

Comment:
Is it possible or expected to have the ZoneId empty but not the Cidr?

My reading of the API spec is that Cidr is optional, not the other way around.